app/src/main/res/values/strings.xml

```diff
@@ -168,10 +168,10 @@
     <string name="request_timeout">Request Timeout</string>
     <string name="superuser_notification">Superuser Notification</string>
     <string name="settings_su_reauth_title">Reauthenticate after upgrade</string>
-    <string name="settings_su_reauth_summary">Reauthenticate superuser permissions after an application upgrades</string>
-    <string name="settings_su_tapjack_title">Enable Tapjacking Protection</string>
+    <string name="settings_su_reauth_summary">Reauthenticate superuser permissions after upgrading apps</string>
```

Comment:
```suggestion
    <string name="settings_su_reauth_summary">Reauthenticate superuser permissions after apps are upgraded</string>
```

Reply:
"Reauthenticate superuser permissions after application upgrades" is the easiest fix to the original string. It's often best to fix these things as simply as possible.

Reply:
I agree. OK.